dev/sparktestsupport/modules.py

```diff
@@ -437,6 +437,7 @@ def __hash__(self):
         "pyspark.sql.pandas.serializers",
         "pyspark.sql.pandas.typehints",
         "pyspark.sql.pandas.utils",
+        "pyspark.sql.observation",
```

Comment:
let's put this in PySpark documentation as well https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.sql.rst

Reply:
so many magic places... done